.github/actions/check-secrets-action/README.md

```diff
@@ -0,0 +1,34 @@
+# Check Secrets Action
+
+This local action runs whenever a change to the `.github/workflows/use-secrets.yml` is pushed to your repository.
+
+There are two inputs necessary for this action to run and provide a grade for this exercise:
+
+| Input           | Description                                                                                                                                                                                                                             | Required parameter |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------: |
+| secrets-context | This a JSON formatted Object containing the [secrets context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts) for the repository from which the `use-secrets.yml` was triggered |         ✅         |
```

Comment:
```suggestion
| `secrets-context` | JSON-formatted object containing the [secrets context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts) for the repository from which the `use-secrets.yml` workflow was triggered |         Yes         |
```

Applying the Microsoft accessibility guidelines regarding the use of emojis.

Reply:
@lucie-docs Object with a capital O is grammatically correct.  In JavaScript an Object is a noun and is almost always referred to with a capital O in any and all documentation you will come across.

Here is a quick look at [the official spec for JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)

Reply:
When it's a class yeah. But I believe in this context you're using it as a noun or am I missing something? 🙂 
This is from the link you sent: "Nearly all objects in JavaScript are instances of `Object`; (...)". First occurrence of object is a noun and not capitalized. Second one refers to the class and is capitalized and between ``. I recommend using the same convention so either it's a class and we write it as `Object` or it's a noun and in this case we write it as object.

Reply:
@lucie-docs fair point.  Small o works for me.  If learners take up issue with it I'm assuming someone will open a PR to fix it.  We get a lot of that with LL courses.